Header header
geometry_msgs/Pose pose
float32 width
float32 score
string shape
string color